### ISSUE TITLE
"xxx-fseg.png" are all black

### ISSUE THREAD

**Issue by 777-en** [bolianchen/pytorch_depth_from_videos_in_the_wild#10]
Hi! Thank you for your nice project. I am trying this work on my own video datasets. I run the following command:
```
CUDA_VISIBLE_DEVICES=5 python gen_data.py \
    --dataset_name video \
    --dataset_dir data/test/video \
    --save_dir data/test/processed \
    --img_height 480 \
    --img_width 640 \
    --mask color \
    --fps 30
```
Then I checked the folder `data/test/processed` and found that all the processed pictures named as "xxx-fseg.png" (for example "0000000000-fseg.png") are all black. But I guess these "xxx-fseg.png"s should mask out the objects that is moving in the scene, right? Additionally, my video is mainly about a man walking on the road, so absolutely there are exactly moving objects in the scene, and I thought they should be segmented and masked out in these "xxx-fseg.png"s.
I'm not sure whether it is abnormal or not? Can you do me some favor? Thank you a lot.

**Comment by bolianchen** [comment]
Hi @777-en
Thanks for pointing out this issue. We apply the Mask RCNN built in torchvision that is pretrained on the COCO dataset. We discard masks of most categories to focus on person and several kinds of vehicles, so it is unexpected for it not being able to detect a man walking. Is it possible to provide me several frames to check this issue? Thanks a lot.

Best,
Bolian

**Comment by 777-en** [comment]
Hi@bolianchen 
Thank you a lot for your kind reply. I've got your idea and I will try to pretrain my own Mask RCNN. Following is a GIF about my video data, here you can see a man walking around. By the way, can you provide me with some examples of "xxx-fseg.png"s which successfully masked out the moving objects? I'd like to see the correct segmentation result. Thank you again for your patience.

![test](https://user-images.githubusercontent.com/73839677/168458212-d8b2f865-abc2-42a3-a6b7-c6680f7cd8a9.gif)

**Comment by bolianchen** [comment]
Hi @777-en 
Thanks for the gif file.
Please check the image and its mask attached below. The mask should be a grayscale image and the pixel values corresponding to possibly moving objects be non-zero. Either painting all the possibly moving objects with 255 or with their respective instance ids is fine.
![n008-2018-08-01-15-16-36-0400__CAM_FRONT__1533151603512404](https://user-images.githubusercontent.com/39755828/168460889-ae630f8e-abea-49db-8f1e-8f0e678a3d52.jpg)
![n008-2018-08-01-15-16-36-0400__CAM_FRONT__1533151604012404-fseg](https://user-images.githubusercontent.com/39755828/168460899-0bd91cbb-a939-4d5e-ac97-5226ff25a7a4.jpg)

Best,
Bolian

**Comment by 777-en** [comment]
Hi @bolianchen
It's so nice to receive your reply, I've got the idea of the mask for moving objects. I will either try to pretrain my own Mask RCNN or perform the manual segmentation. Thank you again!

Best wishes.
